tasks/configure-pipeline-local-fs.yml

```diff
@@ -0,0 +1,140 @@
+---
+
+# Get pipeline uuid
+- name: Get pipeline uuid
+  become: "yes"
+  command: >
+    mysql --user="{{archivematica_src_am_db_user }}"
+          --password="{{ archivematica_src_am_db_password }}"
+          --host="{{ archivematica_src_am_db_host }}"
+          "{{ archivematica_src_am_db_name }}"
+          --batch --skip-column-names
+          --execute="select value from DashboardSettings where name=\"dashboard_uuid\";"
+  register: am_configure_pipelinelocalfs_pipeline_uuid
+
+# Define archivematica_src_configure_pipeline_remote_name
+- set_fact:
+    am_configure_pipelinelocalfs_pipeline_remote_name: "{{ ansible_default_ipv4.address }}"
```

Comment:
I usually prefer ansible_host  , as the default ipv4 address found by ansible is not always reachable

Reply:
Thanks!